jobs/scripts/nightly-builds/nightly-builds.sh

```diff
@@ -96,15 +101,26 @@ case "${CENTOS_VERSION}/${GIT_VERSION}" in
         ;;
 esac
 
+
+case "${CENTOS_VERSION}" in
+     8-stream)
+         MOCK_CHROOT=centos-stream+epel-next-8-x86_64
+     ;;
+     *)
+         MOCK_CHROOT=epel-${CENTOS_VERSION}-${CENTOS_ARCH}
+     ;;
+ esac
+
 # do the actual RPM build in mock
 # TODO: use a CentOS Storage SIG buildroot
 RESULTDIR=/srv/gluster/nightly/${GITHUB_BRANCH}/${CENTOS_VERSION}/${CENTOS_ARCH}
 /usr/bin/mock \
-    --root epel-${CENTOS_VERSION}-${CENTOS_ARCH} \
+    --root ${MOCK_CHROOT} \
     ${MOCK_RPM_OPTS} \
     --resultdir ${RESULTDIR} \
     --rebuild ${SRPM}
 
+exit 0
```

Comment:
this `exit` is probably a left-over from testing?

Reply:
Yes! Its a exit from testing.